3_visualize/src/generate_spreadsheet.R

```diff
@@ -0,0 +1,17 @@
+library(targets)
+library(xlsx)
+tar_load(p1_get_lakes_huc8_sf)
+
+p1_get_lakes_huc8_sf %>%
+  st_drop_geometry() %>%
+  as.data.frame() %>%
+  distinct() %>%
+  select(lake_w_state, Name, HUC8) %>%
+  rename(`Saline lake` = lake_w_state, 
```

Comment:
Can you add another line in here so that the spreadsheet is sorted? Something like `arrange(desc(lake_w_state, HUC8))`

Reply:
Ideally the rows would be grouped by lake, and then by HUC8